recipes/bio_hansel/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: bio-hansel
+  version: "0.1.0"
+
+source:
+  fn: bio_hansel-0.1.0.tar.gz
+  url: https://pypi.python.org/packages/36/21/07f9c1e48794ea712d5496fb58cfea3ca8539f16ea04cf6e2e78c9309560/bio_hansel-0.1.0.tar.gz
+  md5: c7178ecd9e0f30e7669fe497188d7e49
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python >=3.5
```

Comment:
can you remove the pinning here and inlucde under the build section `skip: True  # [py2k]`

Reply:
@bgruening Thanks for reviewing my changes! When you mention `pinning` do you mean the `>=3.5` part?

Reply:
Oh, yes :)

Reply:
Awesome, I'll remove `>=3.5`. Thanks @bgruening.